managed/ui/src/components/universes/UniverseForm/UniverseFormContainer.js

```diff
@@ -79,6 +79,10 @@ const mapDispatchToProps = (dispatch) => {
         dispatch(getTlsCertificates()).then((response) => {
           dispatch(getTlsCertificatesResponse(response.payload));
         });
+        if (response.error) {
+          const errorMessage = response.payload?.response?.data?.error || response.payload.message;
```

Comment:
1. `response.payload.response.data` why we are getting JSON like that?
2. And why `data.error` we can check the status code and populate the `data.message` on top of it.

I believe `response.payload.data` is enough. Can we ask BE folks to structure it properly or is there any particular reason behind that ??

Reply:
@nishantSharma459 unfortunately, that's a flaw of the current way of making API calls

the `response.payload` part is not an actual network response, but smth what a redux action creator returns:
https://github.com/yugabyte/yugabyte-db/blob/c8a558eca9635c73c4c64650d83d3f6d12ba32c3/managed/ui/src/actions/universe.js#L127-L130
then `response.data` is an actual network response as per axios [response schema](https://axios-http.com/docs/res_schema)

so when API returns `{ error: 'bla-bla-bla' }` - above is the way to extract that message from response

fallback to generic `response.payload.message` is needed for cases when API doesn't return any message, like when it's 500 server error of some network connectivity issues

Reply:
Understood.